### PR TITLE
Wfcore 3892 jbossws java args function

### PR DESCRIPTION
**Pull request by rsearls**
https://issues.jboss.org/browse/WFCORE-3892

### CODE REVIEW

**Comment by rsearls** [comment]
That is incorrect.  This PR *must*  be merged first.   The changes in wildfly/wildfly#11114 are dependent in this.

**Comment by marekkopecky** [comment]
> This PR must be merged first. The changes in wildfly/wildfly#11114 are dependent in this.

I know. But wildfly/wildfly#11114 doesn't work even with this PR dependency, so this PR may be wrong (see wildfly/wildfly#11114 for more details)

**Comment by rsearls** [comment]
Don't do any testing of  wildfly/wildfly#11114  until the following 3 WFCORE PRs have been merged.
WFCORE-3830, https://github.com/wildfly/wildfly-core/pull/3333; 
WFCORE-3825, https://github.com/wildfly/wildfly-core/pull/3334;
WFCORE-3892, https://github.com/wildfly/wildfly-core/pull/3327;

**Comment by jmesnil** [comment]
holding this PR as it is not working with wildfly/wildfly#11114 according to https://github.com/wildfly/wildfly/pull/11114#issuecomment-394325385

**Comment by rsearls** [comment]
As I noted previously in this PR discussion this PR is REQUIRED in order for wildfly/wildfly#11114 to running
successfully.   Can we move this forward please.

**Comment by jmesnil** [comment]
@rsearls I saw your comment but @marekkopecky is stating in [his comment](https://github.com/wildfly/wildfly/pull/11114#issuecomment-394325385) that https://github.com/wildfly/wildfly/pull/11114 is not working with *this* PR. Could you check with him please?

Maybe you could setup a custom run on the WildFly CI to test your WFCORE-3892-jbossws-java-args-function branch with the WFLY-10209-appclient-scripts-jboss-modules branch and uses this to validate that everything is working as expected.

**Comment by rsearls** [comment]
I have changed this PR such that it is NOT dependent on 
WFCORE-3830, #3333;
WFCORE-3825, #3334;
WFCORE-3892, #3327;

The changes in this PR are used by code changes present in wildfly/wildfly#11114

File: core-feature-pack/src/main/resources/content/bin/common.ps1 [hunk 1]
```diff
@@ -167,6 +167,74 @@ Param(
   return $PROG_ARGS
 }
 
+# JBossws-cxf utilites require additional input args to be passed to the pgm
+Function Jbossws-Get-Java-Arguments {
```
Comment (jamezp) [inline core-feature-pack/src/main/resources/content/bin/common.ps1]: Besides the addition of the `[string]$javaOpts` what's the difference between this and `Get-Java-Arguments`?

Comment (rsearls) [reply core-feature-pack/src/main/resources/content/bin/common.ps1]: There are 2 input arg differences [string]$scriptName and [string]$javaOpts.   JAVA_OPTS is handled differently in Jbossws-Get-Java-Arguments.

**Comment by jamezp** [comment]
retest this please

**Comment by marekkopecky** [comment]
@jamezp This PR adds `Jbossws-Get-Java-Arguments` and `Set-Java-HotSpot-Option` PowerShell functions. But WF PR https://github.com/wildfly/wildfly-core/pull/3334 (that can use these functions) is closed. Can you close this PR as well? And what is your plan about [WFCORE-3892](https://issues.jboss.org/browse/WFCORE-3892) jira? This jira is not tracked as a bug, but it is tracked as a enhancement.

**Comment by jamezp** [comment]
Thanks for catching that @marekkopecky

**Pull request closed without merging**